test/lunatech/lunchplanner/persistence/MenuPerDayTableSpec.scala

```diff
@@ -14,19 +14,19 @@ object MenuPerDayTableSpec extends Properties(name = "MenuPerDay") with Property
 
   import lunatech.lunchplanner.data.TableDataGenerator._
 
-  override def afterAll(): Unit = dbConnection.db.close()
-
   private val minDate = 0
   private val maxDate = 99999999999999L
 
 
   property("add a new menu per day") = forAll { (menu: Menu, menuPerDay: MenuPerDay) =>
+    createTestSchema()
+
     Await.result(MenuTable.add(menu), defaultTimeout)
 
     val menuPerDayToAdd = menuPerDay.copy(menuUuid = menu.uuid)
     val result = Await.result(MenuPerDayTable.add(menuPerDayToAdd), defaultTimeout)
 
-    cleanMenuPerDayTable
+    dropTestSchema()
```

Comment:
You can move these to `after` (not sure about the exact name, but it's something like that)

Reply:
Same as previous comment